web/src/SidebarItemView.tsx

```diff
@@ -179,6 +185,14 @@ let SidebarItemTimeAgo = styled.span`
   align-items: center;
 `
 
+let SidebarItemActions = styled.div`
+  position: absolute;
```

Comment:
ditto here re: using absolute positioning in a flexbox

Reply:
Fixing this too!